pandas/core/indexes/multi.py

```diff
@@ -277,16 +285,26 @@ def _verify_integrity(self, codes=None, levels=None):
                 raise ValueError("Unequal code lengths: %s" %
                                  ([len(code_) for code_ in codes]))
             if len(level_codes) and level_codes.max() >= len(level):
-                raise ValueError("On level %d, code max (%d) >= length of"
-                                 " level  (%d). NOTE: this index is in an"
-                                 " inconsistent state" % (i, level_codes.max(),
-                                                          len(level)))
+                raise ValueError("On level {level}, code max ({max_code})"
```

Comment:
you can do like this
```
msg = "On level...........".format(....)
raise ValueError(msg)
```
(and if needed make the message nicely formatted over multiple lines, with parens e.g.
```
msg = ("........"
             ".......".format)
```

Reply:
formatting updated